cdc/processor/pipeline/table.go

```diff
@@ -154,15 +169,17 @@ func NewTablePipeline(ctx context.Context,
 		cancel:      cancel,
 	}
 
+	flowController := common.NewTableFlowController(perTableMemoryQuota)
```

Comment:
The memory controller is processor level, which means each changefeed has its own memory quota, is this by design?

Reply:
This is by design. It is actually at the table level. A global solution is not of high priority for now.

Reply:
why it is table level, does all tables in a processor share the same flow controller?

Reply:
The flow controller is actually per-table. Each table has its own quota. This is enough for our purpose now because we have yet to encounter any user who experienced OOM due to large amount of data on a large number of tables.

Sharing the quota among the tables will create very complex deadlock hazards, whose safety would require extensive proofs to demonstrate.